navit/maptool/CMakeLists.txt

```diff
@@ -1,16 +1,41 @@
-
 if(BUILD_MAPTOOL)
+
+	find_package(Protobuf-c REQUIRED)
```

Comment:
This should be in the main CMakeLists.txt and it it is not found disable the maptool build, and it should even fail if manually specified that maptool should be build

Reply:
i could look that but it's bad idea to centralise in the main CMakeLists.txt, things should be closest to what they interfere with, the big big main make things harder to read and improve.